airbyte-api/src/main/openapi/config.yaml

```diff
@@ -24,7 +24,7 @@ info:
     url: "https://opensource.org/licenses/MIT"
 externalDocs:
   description: Find out more about Airbyte
-  url: "https://airbyte.io"
```

Comment:
FYI, this updates the swagger url. The only place this is used in connectors repo is to feed Octavia CLI. So now Octavia CLI will use updated url. just wanted to point out the app implication here, though it is a change we want.

Reply:
Okay I'll remove this